src/config/config.h

```diff
@@ -105,6 +105,8 @@ struct Config {
   bool slave_serve_stale_data = true;
   bool slave_empty_db_before_fullsync = false;
   int slave_priority = 100;
+  int replication_connect_timeout_ms = 3200;
```

Comment:
Connect is 3100 and receive is 3200. It may not be a big issue here but for convenience. 
BTW, you can make them constants since you repeated them twice (and made an error here).